jean_zay/submitit/understanding-unets/eval_only_unet_training.py

```diff
@@ -0,0 +1,47 @@
+from learning_wavelets.training_scripts.exact_recon_unet_training import train_unet
+from learning_wavelets.evaluation_scripts.exact_recon_unet_evaluate import evaluate_unet
+
+from jean_zay.submitit.general_submissions import train_eval_grid
+
+
+job_name = 'exact_recon_unet_training'
+n_epochs = 100
+batch_size = 8
+base_n_filters = 4
+n_layers = 4 
+non_linearity = 'relu'
+bn = True,
+exact_recon = True,
+run_id = 'ExactReconUnet_4_bsd500_0_55_None_1620730822'
+n_gpus = 2
```

Comment:
you only need one gpu for evaluation
```suggestion
n_gpus = 1
```